R/use_r_workflows.R

```diff
@@ -188,3 +188,45 @@ use_spell_check <- function(workflow_name = "call-spell-check.yml") {
   path_to_yml <- file.path(".github", "workflows", workflow_name)
   return(path_to_yml)
 }
+
+#' use workflow to publish to Posit Connect from a private repo
+#'
+#' This action publishes to connect using the git-backed workflow. Note that
+#' it is only necessary to do it this way if the product is in a private or
+#' internal repository; otherwise, it is far easier to deploy following the
+#' instructions in https://docs.posit.co/connect/user/git-backed/
+#' @details To use this workflow, the user will need to generate the manifest.json, create
```

Comment:
Suggestion: maybe delete "create an api key from connect" because "add 2 secrets" implies that users need to create an api key from connect.